java/mdsobjects/src/test/java/MDSplus/MdsTreeTest.java

```diff
@@ -248,9 +256,13 @@ public void testData(){
 		  Assert.fail("Tag test_tag found even if removed");
 		}catch(Exception exc){}
 
+		tree.write();
```

Comment:
In the comment you claim you want to test if the node has been written even though you did not write. instead of writing you should try to getNode after reopening a few lines down and check if it is gone.

Reply:
This might be wrong indeed we actually were thinking about fixing the memory leak